src/sagemaker/workflow/_utils.py

```diff
@@ -225,6 +226,7 @@ def __init__(
         compile_model_family=None,
         description=None,
         depends_on: List[str] = None,
+        container_def_list=None,
         **kwargs,
     ):
         """Constructor of a register model step.
```

Comment:
nit - see below that `model_data` is missing a type in the argument description. i.e. `model_data: the S3...` please add `model_data (str): the S3...`

Reply:
sure i shall update.